scripts/constants.py

```diff
@@ -64,3 +72,38 @@ class ChainId(Enum):
 PRIVATE_KEY = (
     os.environ.get(f"{NETWORK.upper()}_PRIVATE_KEY") or os.environ["PRIVATE_KEY"]
 )
+
+
+def pull_deployments():
```

Comment:
we could move this function to another file because it's not a _constant_, either in the `__init__` or in a `scripts.artifacts` and move there the part from check_resources as well